site/src/pages/TemplateSettingsPage/TemplateGeneralSettingsPage/TemplateSettingsForm.tsx

```diff
@@ -230,21 +233,32 @@ export const TemplateSettingsForm: FC<TemplateSettingsForm> = ({
 								size="small"
 								id="use_classic_parameter_flow"
 								name="use_classic_parameter_flow"
-								checked={form.values.use_classic_parameter_flow}
+								checked={!form.values.use_classic_parameter_flow}
 								onChange={form.handleChange}
 								disabled={false}
 							/>
 						}
 						label={
 							<StackLabel>
-								Use classic workspace creation form
+								<span className="flex flex-row gap-2">
+									Enable dynamic parameters for workspace creation
+									<FeatureStageBadge contentType={"beta"} size="sm" />
+								</span>
 								<StackLabelHelperText>
-									<span>
-										Show the original workspace creation form and workspace
-										parameters settings form without dynamic parameters or live
-										updates. Recommended if your provisioners aren't updated or
-										the dynamic form causes issues.
-									</span>
+									<div>
+										The new workspace form allows you to design your template
+										with new form types and identity-aware conditional
+										parameters. The form will only present options that are
+										compatible and available.
+									</div>
+									<Link
+										className="text-xs"
+										href={docs(
+											"/admin/templates/extending-templates/parameters#dynamic-parameters-early-access",
```

Comment:
```suggestion
											"/admin/templates/extending-templates/parameters#dynamic-parameters-beta",
```

(assuming we merge that change at the same time since this one has a `beta` tag)